src/pages/Company/components/Card.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+function Card({ data }) {
+  const { id, name, compensation } = data;
+  return (
```

Comment:
```suggestion
  const { id, name, compensation } = data;

  return (
```
다른 문맥들끼리는 공백을 통해서 가독성에 신경써주세요